client/package.json

```diff
@@ -16,7 +16,7 @@
     "test": "jest --config ./jest.config.js --maxWorkers=50% --env=jsdom",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "generate": "graphql-codegen --config codegen.yml",
+    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml",
```

Comment:
it's a bit slow if you have to build, like I did due to switching between pg & sqlite.
Useful to not require an external server though..

Reply:
Yeah, I was hoping we would commit `schema.graphql` everytime schema changes